### DISCUSSION TITLE
vue template type

### DISCUSSION THREAD

**Discussion by YWD** [vuejs/pinia#1260] [Help and Questions]
![image](https://user-images.githubusercontent.com/8605395/165447502-8dab3d74-7da8-43c6-a5cd-bf81b43ebd09.png)

I want to know is there way to let template know the type of mainStore?

**Comment by YWD** [comment]
![image](https://user-images.githubusercontent.com/8605395/165447798-84be5fb4-db56-4795-8c11-b1437626aaa1.png)

**Comment by dreambo8563** [comment]
works in vscode  with volar

<img width="434" alt="image" src="https://user-images.githubusercontent.com/6948318/167237047-e792fbdf-9f42-476d-af0e-a581a50cef36.png">
